TypeScript.YAML-tmLanguage

```diff
@@ -169,7 +160,7 @@ repository:
       '5': { name: keyword.operator.ts }
       '6': { name: entity.name.function.ts }
       '7': { name: keyword.operator.ts }
-    end: '(?=\}|;|,|$)|(?<=\})'
```

Comment:
Not it would not be fine. The block could be reorganized as an object body, and it would be hard to differentiate these two cases:
```ts
abstract class C {
    abstract foo() :
    {// object return type
    }
}
```
and 
```ts
abstract class C {
    abstract foo()
    {// method implementation
    }
}
```